app/src/lib/remote-parsing.ts

```diff
@@ -26,6 +26,7 @@ export function parseRemote(url: string): IGitRemoteURL | null {
     new RegExp('^https?://(?:.+@)?(.+)/(.+)/(.+?)(?:.git/?)?$'),
     new RegExp('^git@(.+):(.+)/(.+?)(?:.git)?$'),
     new RegExp('^git:(.+)/(.+)/(.+?)(?:.git)?$'),
+    new RegExp('^ssh://git@(.+)/(.+)/(.+?)(?:.git)?$'),
```

Comment:
Should we do the same fix as https://github.com/desktop/desktop/pull/2610 too? Probably for all the patterns, just to be safe.

Reply:
Yeah, good shout! ✍️